# Fixed processing the last entry in the scan list of the EIT scanner.
#
--- a/eitscan.c	2015/09/10 11:05:03	5.0
+++ b/eitscan.c	2019/03/12 11:46:05
@@ -177,7 +177,7 @@
                       }
                   }
                }
-           if (!scanList->Count() || !AnyDeviceSwitched) {
+           if (!AnyDeviceSwitched) {
               delete scanList;
               scanList = NULL;
               if (lastActivity == 0) // this was a triggered scan
